### DISCUSSION TITLE
is it possible to add ad blocking in Arm/arm64 builds? this is implemented in Cromite

### DISCUSSION THREAD

**Discussion by ghost** [Alex313031/Thorium-Android#2] [Ideas]
it would be very helpful 
Cromite link - https://github.com/uazo/cromite

**Comment by gz83** [comment]
Thanks for your suggestion, we will consider it

cc：Alex313031

@Alex313031

**Reply by ghost** [reply]
Still considering?

**Reply by gz83** [reply]
As long as we don't explicitly object, that means there's still an opportunity to add the features you mentioned.
